index.html

```diff
@@ -19,21 +19,30 @@
             <h2 class="logo"><strong>My Logo</strong></h2>
         </a>
         <nav>
-        <a class="icon">
-            <i class="fa fa-bars"></i>
-        </a>
+        <button class="icon" id="icon">
+            <div class="bar">
+                
+            </div>
+        </button>
         </nav>
-        <ul class="menu">
+        <ul class="menu" >
             <li>Portfolio</li>
             <li>About</li>
             <li>Contact</li>
         </ul>
     </nav>
-    </header>
+        <nav id="menuAll">
+        <ul>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+        </nav>
```

Comment:
- You currently have more than one `nav` tag in your portfolio, to help keep your project more semantically correct, I suggest that you have just one `nav` tag in your HTML page and style your nav using CSS media-queries to fit into either desktop or mobile screens. Please fix this 😄

Reply:
Fixed! 🔧 
The tag from the menu was changed.